make-it/EmMakefile

```diff
@@ -98,6 +98,7 @@ EM_WRAP = --pre-js $(PREJS) --post-js $(POSTJS)
 
 EM_EXPORTS = -s EXPORTED_FUNCTIONS="[\
     '_Vireo_Version', \
+	'_Vireo_MaxExecWakeUpTime', \
```

Comment:
Fix space, it appears not aligned with the other lines on my view.

Reply:
👍